ddtrace/contrib/trace_utils.py

```diff
@@ -288,7 +289,11 @@ def set_http_meta(
         span._set_str_tag(http.QUERY_STRING, query)
 
     if request_headers is not None and integration_config.is_header_tracing_configured:
+        """We should store both http.<request_or_response>.headers.<header_name> and http.<key>. The last one
+        is the DD standardized tag for user-agent"""
         _store_request_headers(dict(request_headers), span, integration_config)
+        if request_headers.get("user-agent", False):
+            span.set_tag(http.USER_AGENT, request_headers["user-agent"])
```

Comment:
I would do:

```suggestion
        if "user-agent" in request_headers:
            span.set_tag(http.USER_AGENT, request_headers["user-agent"])
```

Or:

```suggestion
        user_agent = request_headers.get("user-agent")
        if user_agent:
            span.set_tag(http.USER_AGENT, user_agent)
```

The second might be preferred to avoid the double look-up in the dict, but "key in dict" is pretty quick check so I am ok with whatever we think is easier to read.